circonusapi/circonusdata.py

```diff
@@ -0,0 +1,280 @@
+"""
+Circonus Data Fetching API
+
+This module provides simplified methods for data fetching operations.
+To gain access to the full functionality use the circonusapi module.
+"""
+
+import math
+from datetime import datetime
+
+from circonusapi import circonusapi
+from circllhist import Circllhist
```

Comment:
Probably need to add `circllhist` to `setup.py`, since it is not an optional dependency.

Reply:
I would like to keep the dependency optional. `circonusapi.py` if fully functional without circllhist. Only circounsdata is affected.